src/pages/o/[orgId]/events.tsx

```diff
@@ -52,12 +54,19 @@ const OrgEventsPage : PageWithLayout<OrgEventsPageProps> = (props) => {
     const { orgId } = props;
     const eventsQuery = useQuery('events', getEvents(orgId));
     const orgQuery = useQuery(['org', orgId], getOrg(orgId));
+    const responseQuery = useQuery('eventResponses', getEventResponses);
+    const eventResponses = responseQuery.data;
+
+    const onEventResponse = useOnEventResponse();
```

Comment:
Looking at this again, I think this should all be included in the hook, like I mentioned yesterday. Maybe the hook should even be called `useEventResponses()` (plural) and be used like so:

```typescript
const { eventResponses, onEventResponse } = useEventResponses();
```

Do you feel like you could easily make this work, or should we postpone it to when I can help out? We could merge without it.

Reply:
Yes I agree! It became very clear when I applied `useOnEventResponse()` to [eventId] page yesterday. I believe I can manage with this direction. I'll give it a few hours and if it feels too hard I'll let you know and we merge.